.env.sample

```
@@ -0,0 +1,20 @@
+#MYSQL
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=11111111
+MYSQLDB_DATABASE=book_store_docker
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+SHOW_SQL=true
+HIBERNATE_DDL_AUTO=validate
+
+#Spring
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+
+#JWT
+JWT_SECRET=mySuperSecretForJwtTokenAvailableFromApplicationProperties
+JWT_EXPIRATION=60
+
+#SWAGGER
+DISABLE_SWAGGER_DEFAULT_URL=true
+SWAGGER_PATH=/api/swagger
```

please keep blank line at the EOF at the all files
```suggestion
SWAGGER_PATH=/api/swagger

```